jvb/src/test/kotlin/org/jitsi/videobridge/ConferenceTest.kt

```diff
@@ -16,74 +16,55 @@
 package org.jitsi.videobridge
 
 import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.IsolationMode
 import io.kotest.matchers.shouldBe
-import io.mockk.every
 import io.mockk.mockk
-import io.mockk.mockkStatic
 import org.jitsi.ConfigTest
-import org.jitsi.videobridge.octo.OctoRelayService
-import org.jitsi.videobridge.octo.OctoRelayServiceProvider
-import org.jitsi.videobridge.octo.singleton
+import org.jitsi.videobridge.octo.singleton as octoRelayServiceProvider
 import org.json.simple.JSONObject
 import org.json.simple.parser.JSONParser
 import org.jxmpp.jid.impl.JidCreate
-import org.osgi.framework.BundleContext
 
 /**
  * This is a high-level test for [Conference] and related functionality.
  */
 class ConferenceTest : ConfigTest() {
-    private val octoRelayService = OctoRelayService()
-    private val octoRelaySupplier: OctoRelayServiceProvider = mockk() {
-        every { get() } returns octoRelayService
-    }
-
-    private val bundleContext = mockk<BundleContext> {
-        every { registerService(any() as String, any(), any()) } returns null
-    }
-    private val videobridge = mockk<Videobridge> {
-        every { bundleContext } returns this@ConferenceTest.bundleContext
-    }
+    // The octo relay binds on a port
```

Comment:
Wouldn't it be easier to keep the mock for octo?

Reply:
It was a mock for the `OctoRelay*Supplier*` with a real `OctoRelayService` instance. The tentacle needs the UDP and Conf transports, so there would be a whole bunch of things to mock.